### ISSUE TITLE
Style trove classifier checkboxes

### ISSUE THREAD

**Issue by dstufft** [pypi/warehouse#754]
The checkboxes for the trove classifiers on the search/browse page are currently unstyled and despite indentation, it is hard to see which classifiers are children of other classifiers:

![ca5a7c4e-871f-11e5-8ab5-2b1aaa918ea0](https://cloud.githubusercontent.com/assets/145979/11277989/fbfbbbc0-8eb6-11e5-9795-183b07e8547b.png)

I propose restyling the checkboxes to match the rest of the UI, and using line connectors to indicate parent/child relationships:

![3e4e544a-8720-11e5-8a52-68c9f882fd1f](https://cloud.githubusercontent.com/assets/145979/11277998/05cf6d90-8eb7-11e5-8a04-e54c97fe9cf5.png)

Please also note that the checkbox should be left aligned directly under the open/close title icon.

We will also need to make some decisions regarding whether or not children classifiers should be selected by default when a parent is selected.

_Copied from nlhkabu/warehouse-ui#1_

**Comment by nlhkabu** [comment]
@dstufft are you able to look into potential JS plugins for this?  I am pretty sure we will not be able to do it with CSS alone...  Thanks :D

**Comment by nlhkabu** [comment]
While we are at it, we need to change our markup from:

```html
<label><input type="checkbox">My Label</label>
```
To:
```html
<input type="checkbox" id="mycheckbox"><label for="mycheckbox">My Label</label>
```

**Comment by nlhkabu** [comment]
I can live with these unstyled for launch, so I'm going to change the milestone.

**Comment by nlhkabu** [comment]
Update on this- we can do this with pure CSS, so no need for a JS plugin:
http://odyniec.net/articles/turning-lists-into-trees/

However, this will require us to restructure the HTML to use nested lists.

**Comment by di** [comment]
@nlhkabu We would still need to preserve the checkboxes too I think. Would that approach still work?

**Comment by nlhkabu** [comment]
Hi @di - yes that would still work.  In the short term, it would be good to keep the standard checkboxes, and just draw the lines with CSS under them.

**Comment by nlhkabu** [comment]
I have opened a separate issue for the creation of a tree (see #1299). I will leave this issue open as we will also need to style the actual checkboxes to match the tree style.

**Comment by miketheman** [comment]
Closing, as this was implemented at some point, likely in #3273. The current visual looks like:

<img width="173" alt="Screenshot 2022-12-19 at 11 34 12 AM" src="https://user-images.githubusercontent.com/529516/208474478-ea47b840-520a-40c2-8e82-0339bcf58019.png">

**Comment by di** [comment]
I think this was left open because the checkboxes are still unstyled, but IMO that is probably fine and maybe even preferable at this point.
